UnityProject/Assets/Scripts/Mobs/Traversal/MobTraversal.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using HealthV2;
+using InGameGizmos;
+using Logs;
+using Tilemaps.Behaviours.Pathfinding;
+using UnityEngine;
+
+namespace Mobs.Traversal
+{
+	public class MobTraversal : MonoBehaviour
+	{
+		public PlayerScript Mob;
+		public MovementSynchronisation Movement;
+		public int MaxQueuedTargets = 12;
+		public int MaxRetries = 6;
```

Comment:
What is the reasoning for 6? I feel if a pathfinding attempt has already failed twice its already reason enough to move to the next target in the queue, unless I am mis interpreting what these retries are for

Reply:
There could be a problem during their traversal that doesn't let them reach their next tile, so they need to retry traversal again to `pos` in case they encounter something like a table, door, a player swapping with them, a player pulling them, atmos throwing them around, etc.

If they stopped immediately after the second or third attempt, then they would look like they give up easily, or they lost track of what they're doing mere seconds after they are given a goal. 6 is a sweet spot between giving them time to perform actions (like attacking a tile or waiting for something to open) while also not making them look like they are stuck in place.